doc/input-output-reference/src/overview/group-system-availability-managers.tex

```diff
@@ -600,6 +600,15 @@ \subsubsection{Inputs}\label{inputs-10-021}
 
 \textbf{Note:} A \textbf{ZoneInfiltration:*} object indicates any one of \textbf{ZoneInfiltration:DesignFlowRate}, \textbf{ZoneInfiltration:EffectiveLeakageArea},and \textbf{ZoneInfiltration:FlowCoefficient} objects.A object of\textbf{ZoneVentilation:*} indicates any one of \textbf{ZoneVentilation:DesignFlowRate} and \textbf{ZoneVentilation:WindandStackOpenArea} objects.
 
+\paragraph{Field:Minimum HVAC Operation Time}\label{field-minimum-hvac-operation-time}
+
+ The field represents the minimum time that HVAC system operation will be allowed. If accumulated system operation time is less than this value, the system operation status will be forced regardless of other operation conditions, and natural ventilation will shut off. 
+
+\paragraph{Field:Minimum Ventilation Time}\label{field-minimum-ventilation-time}
+
+ The field represents the minimum time that natural ventilation will remain. If accumulated ventilation time is less than this value, the natural ventilation will be forced regardless of other operation conditions. 
```

Comment:
@mjwitte Good wording. I use the latter. I also made similar changes in the Minimum HVAC Operation Time field.